docker_build

```diff
@@ -79,5 +86,5 @@ docker buildx build \
   --label="LIBPCAP_VERSION=${LIBPCAP_VERSION}" \
   --label="TCPDUMP_VERSION=${TCPDUMP_VERSION}" \
   --label="PCAPCLI_VERSION=${PCAPCLI_VERSION}" \
-  --no-cache -t "${2}-${DOCKER_TAG_SUFFIX}_pcapcli-${PCAPCLI_VERSION}" \
-  $(pwd)
+  --no-cache -t "${2}:${3}-${1: -4}" -t "${2}:${$IMAGE_TAG}" \
```

Comment:
we may want to make this more flexible to allow for more envs such as GAE or GKE;
for now this approach works great for Cloud Run.